rapid_pro/synchronise_contacts.py

```diff
@@ -119,7 +119,7 @@ def filter_valid_contacts(contacts):
                 log.warning(f"Found a contact with multiple URNS; skipping. "
                             f"The RapidPro UUID is '{contact.uuid}'")
                 continue
-            if contact.urns[0].startswith("tel:") and not contact.urns[0].startswith("tel:+"):
+            if contact.urns[0].startswith("tel:") and not contact.urns[0].startswith("tel:+254"):
                 log.warning(f"Found a contact with a telephone number but without a country code; skipping. "
                             f"The RapidPro UUID is '{contact.uuid}'")
```

Comment:
If you do this here you'll need to update the warning message too. 

It might be worth separating this check so we can distinguish no country code from no +254 code, or rewriting to reject numbers in the form +10... or +11... with a length check, so that we can merge this and have this check available in future across all projects.

Reply:
yes, yes - have refactored this to check for contacts that startswith '+1' are less than 11 characters long. 
Other factors considered:
- US country code is +1 but the character length greater than 11. 
- Golis contacts are stored as 256 and not +256 therefore we can't use 12 to validate telephone number.

Reply:
Thanks. I can't think of any other cases we need to consider, but not concerned because this fails safe (in other words, if we overmatch we'll see the warning).